### PR TITLE
Fix various things to comply with the style guide.

### PR DESCRIPTION
**Pull request by thealmarty**
Just a quick/non-controversial thing to do which brings us closer to style guide compliant.

- fix line length
- removed explicit imports if they are too long, as per [style guide](https://github.com/input-output-hk/plutus/blob/master/STYLEGUIDE.adoc#imports).
- remove some unused pragmas.
- remove some redundant brackets.

Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Changelog fragments have been written (if appropriate)
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, PNG optimization, etc. are updated
- PR
    - [ ] (For external contributions) Corresponding issue exists and is linked in the description
    - [x] Targeting master unless this is a cherry-pick backport
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested

### CODE REVIEW

File: doc/notes/fomega/cek-cps-experiments/src/CekMachine.cps.hs [hunk 1]
```diff
@@ -1,5 +1,3 @@
--- editorconfig-checker-disable-file
-
 -- This file contains a version of the CEK machine in
```
Comment (michaelpj) [inline doc/notes/fomega/cek-cps-experiments/src/CekMachine.cps.hs:1]: `doc/notes/fomega` should probably be excluded wholesale, or probably just deleted tbh.

File: doc/read-the-docs-site/tutorials/AuctionValidator.hs [hunk 2]
```diff
@@ -9,7 +8,7 @@ import PlutusCore.Default qualified as PLC
 import PlutusLedgerApi.V1 (POSIXTime, PubKeyHash, Value, adaSymbol, adaToken, singleton)
 import PlutusLedgerApi.V1.Address (pubKeyHashAddress)
 import PlutusLedgerApi.V1.Interval (contains)
-import PlutusLedgerApi.V2 (Datum (..), OutputDatum (..), ScriptContext (..), TxInfo (..), TxOut (..), from, to)
```
Comment (michaelpj) [inline doc/read-the-docs-site/tutorials/AuctionValidator.hs:12]: I don't think this is too long? It just needs to be split over multiple lines.

Comment (michaelpj) [reply doc/read-the-docs-site/tutorials/AuctionValidator.hs:12]: I think there's an ambiguity: I don't think the intention was that "long import statement" meant "the import statement hits the line length", but rather "the import statement is annoyingly long".

**Review by michaelpj — APPROVED** [review]

**Review by kwxm — APPROVED** [review]

File: doc/read-the-docs-site/tutorials/AuctionValidator.hs (hunk 2 above)
Comment (thealmarty) [reply doc/read-the-docs-site/tutorials/AuctionValidator.hs:12]: I edited our stylish-haskell config. It seems the "column" config only applies to imports. Also, for those who cases, we can configure how the imports that are too long are broken up: https://github.com/haskell/stylish-haskell/blob/13f1db77f28b62ac6da6abb3c82244d5e740503b/data/stylish-haskell.yaml#L198

File: doc/notes/fomega/cek-cps-experiments/src/CekMachine.cps.hs (hunk 1 above)
Comment (effectfully) [reply doc/notes/fomega/cek-cps-experiments/src/CekMachine.cps.hs:1]: Why would we delete `doc/notes/fomega`? It contains the entire history of us choosing Scott-encoded data types, `ifix`, mutually recursive functions and whatnot. I think it's useful to have this history, even though it's partially duplicated in the papers. And it has low to no maintenance costs.

**Pull request merged**